### PR TITLE
JPN-670 Truncate discussions text to 3 lines

### PR DESCRIPTION
**Pull request by perjg**
Make sure content for discussions cards are clamped at 3 lines.

Based on the same code from Mercury:
- CSS for webkit based browsers
- JS workaround for FF / Edge.

@Grunny @garthwebb

### CODE REVIEW

File: extensions/wikia/EmbeddableDiscussions/scripts/ext.embeddableDiscussions.js [hunk 1]
```diff
@@ -64,18 +64,24 @@ require([
 			i,
 			thread,
 			userData,
-			date;
+			date,
+			content;
 
 		for (i in threads) {
```
Comment (WikiaTech) [inline extensions/wikia/EmbeddableDiscussions/scripts/ext.embeddableDiscussions.js:70]: MAJOR Restrict what this loop acts on by testing each property. [rule](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AForIn)

File: extensions/wikia/EmbeddableDiscussions/scripts/ext.embeddableDiscussions.js [hunk 2]
```diff
@@ -116,7 +122,8 @@ require([
 				withCredentials: true
 			},
 		}).done(function (data) {
-			var threads = processData(data._embedded.threads, requestData.upvoteRequestUrl);
+			var threads = processData(data._embedded.threads, requestData.upvoteRequestUrl),
+				i;
```
Comment (WikiaTech) [inline extensions/wikia/EmbeddableDiscussions/scripts/ext.embeddableDiscussions.js]: MAJOR Remove the declaration of the unused 'i' variable. [rule](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AUnusedVariable)

File: extensions/wikia/EmbeddableDiscussions/scripts/templates.mustache.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
 /* jshint ignore:start */ define( 'embeddablediscussions.templates.mustache', [], function() { 'use strict'; return {
```
Comment (WikiaTech) [inline extensions/wikia/EmbeddableDiscussions/scripts/templates.mustache.js:1]: MINOR Reformat the code to have only one statement per line. [rule](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AOneStatementPerLine)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues
* MAJOR 3 major
* MINOR 1 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [ext.embeddableDiscussions.js#L10](https://github.com/Wikia/app/blob/efda4b02ac2ad5949914a13049bfa88718dba19e/extensions/wikia/EmbeddableDiscussions/scripts/ext.embeddableDiscussions.js#L10): Function has 8 parameters which is greater than 7 authorized. [rule](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AExcessiveParameterList)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
- MAJOR 2 major
- MINOR 1 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:
1. MAJOR [ext.embeddableDiscussions.js#L10](https://github.com/Wikia/app/blob/ed1c6357d5e80551bb1277fc5a6bf3819bf2e9be/extensions/wikia/EmbeddableDiscussions/scripts/ext.embeddableDiscussions.js#L10): Function has 8 parameters which is greater than 7 authorized. [rule](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AExcessiveParameterList)

**Review by Grunny — APPROVED** [review]
LGTM.

**Pull request merged**